### PR TITLE
portfolio-website-contact-form

### PR DESCRIPTION
**Pull request by KMT-Media**
During this pull request, the following actions are implemented:
 * forms created with HTML5 validations in the footer.
 * The input fields are designed to accept a specific range of data, based on the requirement given to us.
 * flexbox is used to lay out all elements on this footer.
 * Button interactions

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -444,5 +444,40 @@ <h2 class="primary-text">About Myself</h2>
...
+          cols="30"
+          rows="10"
+          maxlength="500"
+          name="user_message"
+          placeholder="Write your message here..."
+          required
+        ></textarea>
+        <button type="submit" class="form-btn">Get in Touch</button>
+      </form>
+    </footer>
```
Comment (mckent05) [inline index.html]: - To improve the semantics of your page, I suggest that you do not use a `footer` tag to wrap this section of the page. Using a `section` tag would be more semantically correct. Please fix this 👍 👍

File: style.css [hunk 2]
```diff
@@ -248,3 +249,84 @@ li:hover {
...
+  height: 800px;
+  background: url(./assets/images/contact\ form\ shapes-mobile@2x.png);
+  padding: 130px 24px 20px 24px;
+  display: flex;
+  flex-direction: column;
+  gap: 20px;
+  margin-top: 100px;
+  background-color: #6070ff;
+  border-radius: 50px 0 0 0;
+}
```
Comment (mckent05) [inline style.css]: - Good job adding the image using the `background` property 👏 👏 . But it does not display properly on bigger mobile screen.Please reference the image I attached below. I suggest you consider using [background-repeat](https://www.w3schools.com/CSSref/pr_background-repeat.asp) property and `background-position` property to make sure the image is aligned properly.

Your Project:
responsive contact

File: style.css [hunk 3]
```diff
@@ -248,3 +249,84 @@ li:hover {
...
+  height: 35px;
+  color: #6070ff;
+  background: #fff;
+  border: none;
+  border-radius: 8px;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  align-self: center;
+}
```
Comment (mckent05) [inline style.css]: - This button isn't properly aligned as it is in the Figma template. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏. Please reference the image below and fix this 😄 

Your Project:
misaligned button

Design Template:
figma button

**Review by mckent05 — CHANGES_REQUESTED** [review]
Hi @KMT-Media,

# STATUS: CHANGES REQUIRED ♻️ ♻️  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
almost there

## Highlights
- Your contact form is working as expected 👍 👍 
- All linter checks are passing ✔️ 
- PR title and Summary are descriptive 👏 👏 


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (KMT-Media) [reply index.html]: Thankyou @mckent05

File: style.css (hunk 2 above)
Comment (KMT-Media) [reply style.css]: Thankyou @mckent05

File: style.css (hunk 3 above)
Comment (KMT-Media) [reply style.css]: Thankyou @mckent05

**Review by Debas-31 — APPROVED** [review]
> ## Hi team :wave:, 

# Status: Approved :green_circle: 

Great work on making the changes requested by a previous reviewer. Your project is complete !  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights
- Linters setup :heavy_check_mark:
- GitHub flow :heavy_check_mark: 
- Project functiionality :heavy_check_mark: 
- Figma template usage :heavy_check_mark: 

## Optional suggestions
[OPTIONAL] - Kindly ask your partner to contribute a code to this project same you also contribute to his project.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**